teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -152,6 +153,16 @@ public class ValidatorOptions {
   private boolean isLocalSlashingProtectionSynchronizedEnabled =
       DEFAULT_VALIDATOR_IS_LOCAL_SLASHING_PROTECTION_SYNCHRONIZED_ENABLED;
 
+  @Option(
+      names = {"--shut-down-when-validator-slashed-enabled"},
```

Comment:
Generally we have a test showing these enabled / not enabled, sometimes checking default value...
That test is not present here. It would be beneficial to add 2 test cases, 1 showing it being set to true, the other checking default setting.

Reply:
Done here #7915